lightning/src/ln/async_payments_tests.rs

```diff
@@ -872,11 +852,7 @@ fn async_receive_mpp() {
 
 	let amt_msat = 15_000_000;
 	let payment_id = PaymentId([1; 32]);
-	let params = RouteParametersConfig::default();
-	nodes[0]
-		.node
-		.pay_for_offer(&offer, None, Some(amt_msat), None, payment_id, Retry::Attempts(1), params)
```

Comment:
Presumably the different `Retry::Attempts` values used throughout these tests weren't necessary?